### PR TITLE
Fix print links for table section

### LINKED ISSUE #19: Broken URL for Printing Tables section
**I'm submitting a ...**

- [x] bug report
- [ ] feature request
- [ ] other (Please do not submit support requests here (below))

## Bug reports

I'm not able to access to the article by using this URL: http://css-discuss.incutio.com/wiki/Printing_Tables

My proposal is to replace it by an article from the [Smashing Magazine](https://www.smashingmagazine.com):

- Designing For Print With CSS: https://www.smashingmagazine.com/2015/01/designing-for-print-with-css/#page-breaks

I will linked to this issue the corresponding pull request. Hope it helps you. Thanks

### PR DESCRIPTION
**Pull request by antleblanc**
fix #19

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

This pull request doesn't have any tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ At printing time, these styles will:
   that they should:
 
   * ensure the table header (`<thead>`) is [printed on each page spanned by the
-    table](http://css-discuss.incutio.com/wiki/Printing_Tables)
+    table](https://www.smashingmagazine.com/2015/01/designing-for-print-with-css/#page-breaks)
```
Comment (roblarsen) [inline README.md]: I don't think this link covers everything we have here, does it? I wonder if there's something that is closer.

**Comment by roblarsen** [comment]
Thanks! I think fixing this is a good idea. I don't think the article linked covers everything we do. I wonder if there's anything else we can link to in order to bolster this information.

**Comment by antleblanc** [comment]
Thanks @roblarsen for your review.

I will search for other articles that's covers table header and let you decide which one sounds more appropriate in this case.

But what I found quite interesting about the Smashing Magazine article is that it covers multiple aspects about the print CSS.

**Comment by roblarsen** [comment]
That article is interesting and it may be worth looking at the print section with fresh eyes. I just don't think we can swap out that link for the one we're replacing without further context. 

@coliff what do you think about looking at the print section to see if there's something interesting we should be doing?

**Comment by antleblanc** [comment]
Others resources:
- http://www.terminally-incoherent.com/blog/2009/10/12/repeating-html-table-headers-on-each-printed-page/
- https://stackoverflow.com/a/52040631

**Comment by coliff** [comment]
One suggestion for a quick and easy fix for the broken link is to simply link to the same page on the Wayback Machine archive:
https://web.archive.org/web/20180815150934/http://css-discuss.incutio.com/wiki/Printing_Tables

Agree with Rob that it would be worth looking into this again in more depth - that article was last updated in November 2004 (!)

**Comment by roblarsen** [comment]
@antleblanc Do you want to follow @coliff's suggestion and just use the wayback machine for now? I think we can use this issue as a jumping off point to reevaluate the print section  I appreciate the links 👍

**Comment by antleblanc** [comment]
@roblarsen I've applied the @coliff's suggestion.

**Comment by roblarsen** [comment]
👍

**Pull request merged**